src/test/suite/utils/fs.mock.ts

```diff
@@ -0,0 +1,9 @@
+import * as fs from "fs";
+import * as sinon from "sinon";
+
+export default function mockExistsSync(args: [fs.PathLike, boolean][]): void {
```

Comment:
You can use the `...args` syntax to expose a cleaner api: see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments#rest_default_and_destructured_parameters